ancilla.el

```diff
@@ -111,9 +112,61 @@ call `ancilla-rewrite' otherwise."
   (interactive)
   (ancilla--call-adaptor-with-instruction 'rewrite))
 
-;; ------------ PRIVATE FUNCTIONS --------------
+;; shortcuts following intellij copilot plugin
+
+(defun ancilla-simplify-this ()
+  "Simplify the current selection."
+  (interactive)
+  (ancilla--process-rewrite "Simplify this"))
```

Comment:
how about let the user to customize the prompts by prefixing `C-u`?

Reply:
yes. i wish to have a better prompts than this.. for more user specific simplify operation, i think ancilla-rewrite is a better option.

Reply:
Make sense. Let's keep i this way then. Ultimately I think it may be a good idea to let the user to customize the shortcuts.